pkg/upgrade/v1alpha2/kubernetes_test.go

```diff
@@ -679,68 +679,83 @@ func TestUpgradeTaskPatch(t *testing.T) {
 
 func TestGetUpgradeDetailedStatuses(t *testing.T) {
 	tests := map[string]struct {
-		step      apis.UpgradeStep
-		phase     apis.StepPhase
-		args      []string
-		expectErr bool
+		status       apis.UpgradeDetailedStatuses
+		expectOutput bool
 	}{
 		"Test 1": {
-			apis.PreUpgrade,
-			apis.StepWaiting,
-			[]string{},
-			false,
+			apis.UpgradeDetailedStatuses{
```

Comment:
nit: `status: apis.UpgradeDetailedStatuses{}` will be helpful in case of test failure.

Reply:
I think Test 4 covers that

Reply:
No I am saying in below pattern
```go
"Test 1": {
			status:   apis.UpgradeDetailedStatuses{
				           Step: apis.PreUpgrade,
				           Status: apis.Status{
					        Phase: apis.StepWaiting,
				           },
			},
			expectOutput: true,
		},
```
it's UT we can ignore